applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -343,6 +343,12 @@ static void syncable_cb(struct bt_bap_broadcast_sink *sink, bool encrypted)
 
 	LOG_DBG("Broadcast sink is syncable");
 
+	if (sink != broadcast_sink) {
+		broadcast_sink_cleanup();
+		le_audio_event_publish(LE_AUDIO_EVT_SYNC_LOST);
```

Comment:
Does `LE_AUDIO_EVT_SYNC_LOST` relate to the PA sync or BIG sync? 

Neither of them is really what is happening here though, right?

Reply:
Relates to the BIG sync, it's how we can "reset" the system and start scanning again